src/helpers/theme.js

```diff
@@ -1,223 +1,248 @@
-    },
-  },
-  paper: {
+    palette: {
+      primary: { main: primaryColor },
+      secondary: { main: whiteColor },
+      error: { main: errorColor },
+      text: {
+        primary: fontColor,
+        secondary: fontColor,
+        second: whiteColor,
+        error: errorColor,
+      },
+      toggledButton: toggledButtonColor,
+    },
+    typography: {
+      useNextVariants: true,
+      fontFamily: ["Rubik", "Roboto", '"Helvetica Neue"', "sans-serif"].join(","),
+      fontSize: 14,
+      fontWeightRegular: 300,
+      fontWeightMedium: 400,
+      title: {
+        fontSize: 20,
+        fontWeight: 300,
+      },
+      label: {
+        color: greyColor,
+      },
+    },
+    jrnlDrawer: {
+      open: {
+        width: 500,
+      },
+      close: {
+        width: 80,
+      },
+      itemDetail: {
+        marginLeft: 8,
+      },
+      iconSize: 24,
+    },
+    menu: {
+      variant: "AppBar",
+      drawer: {
+        width: 300,
+        fontSize: 16,
+        backgroundColor: primaryColor,
+      },
+      appBar: {
+        fontSize: 16,
+      },
+    },
+    page: {
+      padding: 16,
+      locked: {
+        background: lockedBackgroundPattern,
+      },
+    },
     paper: {
-      margin: 10,
-      backgroundColor: "#dbeef0",
-    },
-    header: {
-      color: "#006273",
-      backgroundColor: "#b7d4d8",
-    },
-    message: {
-      backgroundColor: "#b7d4d8",
-    },
-    title: {
-      padding: 10,
-      fontSize: 24,
-      color: "#006273",
-      backgroundColor: "#b7d4d8",
-    },
-    action: {
-      padding: 5,
-    },
-    divider: {
-      padding: 0,
-      margin: 0,
-    },
-    body: {
-      marginTop: 10,
-      backgroundColor: "#dbeef0",
-    },
-    item: {
-      padding: 10,
-    },
-  },
-  table: {
-    title: {
-      padding: 10,
-      fontWeight: 500,
-      color: "#006273",
-      backgroundColor: "#b7d4d8",
-    },
-    header: {
-      color: "#006273",
-    },
-    headerAction: {
-      padding: 5,
-    },
-    row: {
-      color: "#006273",
-      align: "center",
-      "&:hover": {
-        background: "rgba(0, 0, 0, 0.12) !important",
-      },
-    },
-    cell: {
-      padding: 5,
-    },
-    lockedRow: {
-      background: "repeating-linear-gradient(45deg, #D3D3D3 1px, #D3D3D3 1px, #fff 10px, #fff 10px);",
-    },
-    lockedCell: {},
-    highlightedRow: {},
-    highlightedCell: {
-      fontWeight: 500,
-      align: "center",
-    },
-    secondaryHighlightedRow: {
-      backgroundColor: "#cbedf2",
-    },
-    secondaryHighlightedCell: {},
-    highlightedAltRow: {},
-    highlightedAltCell: {
-      fontStyle: "italic",
-      align: "center",
-    },
-    disabledRow: {},
-    disabledCell: {
-      // textDecoration: "line-through",
-      color: "grey",
-      align: "center",
-    },
-    footer: {
-      color: "#006273",
-    },
-    pager: {
-      color: "#006273",
-    },
-  },
-  form: {
-    spacing: 10,
-  },
-  formTable: {
```

Comment:
why we hadd formTable{ paper and now paper{ paper ?

Reply:
![Screenshot from 2025-05-08 09-58-50](https://github.com/user-attachments/assets/e2ab4db3-94e8-4788-8c46-717958235884)
That's normal. You can see that on develop there is the same situation. The same with formTable
You can take a look into https://github.com/openimis/openimis-fe_js/blob/2e887c7e817c7bf583795488700fadff7f6b73b7/src/helpers/theme.js and compare with develop https://github.com/openimis/openimis-fe_js/blob/develop/src/helpers/theme.js